be/src/common/config.h

```diff
@@ -250,7 +250,7 @@ CONF_Int64(index_stream_cache_capacity, "10737418240");
 // CONF_Int64(max_packed_row_block_size, "20971520");
 
 // Cache for storage page size
-CONF_String(storage_page_cache_limit, "20%");
+CONF_mString(storage_page_cache_limit, "20%");
 // whether to disable page cache feature in storage
 CONF_Bool(disable_storage_page_cache, "false");
```

Comment:
should we support disable/enable page cache without restart?

Reply:
Yes, I think it's a good idea from a product perspective.  Let's open another issue to handle it.

Reply:
https://github.com/StarRocks/starrocks/issues/11190